lib/serializers/holding.js

```diff
@@ -0,0 +1,163 @@
+const sierraLocationMapping = require('@nypl/nypl-core-objects')('by-sierra-location')
+
+const utils = require('../utils')
+const Statement = require('../models/statement')
+const Creator = require('../models/creator')
+const buildFieldMapper = require('../field-mapper')
+
+const log = require('loglevel')
+
+const fromMarcJson = (object) => {
+  try {
+    const fieldMapper = buildFieldMapper('holding', null)
+
+    const builder = Statement.builder(object.id, Creator.CORE_SERIALIZER.id, { record_id: object.id })
```

Comment:
Bib and Item ids are prefixed (the `b` and `i` standard prefixes we all know and love), but generally we don't have a prefix for the holdings records. I'm not sure if it would be a problem if we don't? Otherwise, we would need to add `h` as a prefix to all of the identifiers used here.

I have no real preference other than a desire to keep things simple. I feel like we probably should but want to get input from others first.

Reply:
You're proposing not prefixing holding ids? Everything in the DiscoveryStore exists in the same id-space. All records are statements about "resources". There's thus a technical limitation that whatever id you use be unique in the universe of bib and item ids. Because the latter have used prefixes, you could technically not use prefixes for holdings, but we'd only get to do that for one resource type. Short term, I'd use 'h' as a prefix (or 'holding:'). Medium term, maybe we need a dedicated "resource_statement" table for each type (i.e. bib_statement, item_statement, holding_statement). Long term, the DiscoveryStore and greater data pipeline should be rethought from a high level.

Reply:
ok yeah, I had a feeling that would be the answer after thinking about this more and definitely agree. Was trying to get away something that would have been too simple and almost certainly would have caused issues